api/swagger.yaml

```diff
@@ -5612,6 +5612,13 @@ definitions:
         items:
           type: "string"
         example: ["--debug", "--systemd-cgroup=false"]
+      status:
+        description: |
+          Status of the runtime.
```

Comment:
I think (for a follow-up) we could tweak this description a bit, to outline what to expect here; similar to `driverstatus` https://github.com/moby/moby/blob/f6784595930d75bba835d4358ff9f4c7b9431636/api/swagger.yaml#L4960-L4973

Reply:
done